test/e2e/suite/conformance/certificates/acme/acme.go

```diff
@@ -28,76 +28,72 @@ import (
 	"github.com/jetstack/cert-manager/test/e2e/framework/addon/pebble"
 	"github.com/jetstack/cert-manager/test/e2e/framework/addon/tiller"
 	"github.com/jetstack/cert-manager/test/e2e/suite/conformance/certificates"
+	"github.com/jetstack/cert-manager/test/e2e/suite/issuers/acme/dnsproviders"
 )
 
 var _ = framework.ConformanceDescribe("Certificates", func() {
 	// unsupportedFeatures is a list of features that are not supported by the ACME
 	// issuer type using HTTP01
 	var unsupportedFeatures = certificates.NewFeatureSet(
 		certificates.IPAddressFeature,
-		certificates.Wildcards,
+		certificates.WildcardsFeature,
```

Comment:
Wildcards *are* supported by the DNS01 provider, we should only disable them for HTTP01 😄